signac_dashboard/views.py

```diff
@@ -50,9 +51,21 @@ def jobs_list(dashboard):
     project_title = dashboard.project.config.get("project", None)
     g.title = f"{project_title}: Jobs" if project_title else "Jobs"
     g.subtitle = g.pagination.item_counts()
+    session["context"] = "JobContext"
     return dashboard._render_job_view(default_view="list")
 
 
+def project_info(dashboard):
+    g.project = dashboard.project
+    project_title = dashboard.project.config["project"]
+    schema_version = dashboard.project.config["schema_version"]
+    project_dir = dashboard.project.config["project_dir"]
+    g.title = f"{project_title} - schema version {schema_version}"
+    g.subtitle = f"Project directory: {project_dir}"
```

Comment:
Showing the project directory is not something I would advise for security reasons. If someone wishes to host this on a public server, this is exposing information about the server (usernames, paths) that would be highly undesirable. Additional details shared via Slack private message.